learn.md

```diff
@@ -41,45 +41,45 @@ redirect_from:
 		<!--<h3 id="writing-idiomatic-ballerina-code"><a href="/learn/user-guide/">Writing Idiomatic Ballerina Code</a></h3>
 		<p>Guide developers to start thinking in Ballerina.</p>-->
 		<h3 id="organizing-ballerina-code"><a href="/learn/organizing-ballerina-code/package-layout/">Organizing Ballerina Code</a></h3>
-		<p>Basics of projects/packages/modules.</p>
+		<p>Basics of projects, packages, and modules.</p>
 		<h3 id="testing-ballerina-code"><a href="/learn/testing-ballerina-code/testing-quick-start/">Testing Ballerina Code</a></h3>
-		<p>Write automated tests using the built-in test framework.</p>
+		<p>Writing automated tests using the built-in test framework.</p>
```

Comment:
Shouldn't we change this to `Details on writing automated tests using the built-in test framework`? As other topics in this section started with `Details on` or `Basics of`

Reply:
Changed.